docs/tileentities/tesr.md

```diff
@@ -26,3 +26,29 @@ Registering a TER
 -----------------
 
 In order to register a TESR, call `ClientRegistry#bindTileEntitySpecialRenderer` passing the tile entity class to be renderer with this TER and the instance of the TER to use to render all TEs of this class.
+
+IRenderTypeBuffer - Vertex Builders
+-----------------------------------
+This class gives you access to `IRenderTypeBuffer`, which in turn provides a series of `IVertexBuilder` instances via the `IRenderTypeBuffer#getBuffer` method.
+
+To get an example vertex builder (VB), see the `RenderTypes` class for a set of prebuilt builders.
+
+There are various vertex builders that Minecraft provides by default; here is a small overview of the main VB instances:
+
+| Render Type |  Example Purpose
+| :---------- | :--------------
+| SOLID | Rendering block faces
+| CUTOUT_MIPPED | Rendering parts of a block, or additional, dynamic models
+| CUTOUT | See above, difference is texture sheet usage
+| LINES | Directional lines
```

Comment:
Needs a better description than what's provided. For example, the `LINES` render type can not only draw directional lines, but can also be colored, supports an alpha channel, and targets the item entity framebuffer.